### PR TITLE
Improve description of model section

### PR DESCRIPTION
**Pull request by alexshalamov**
- Improve description of model section
- Add graphical example for relation of Earth's and device's coordinate systems

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -72,9 +91,20 @@ of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
 
 A [=latest reading=] per [=sensor=] of orientation type includes one [=map/entries=]
 whose [=map/key=] is "quaternion" and whose [=map/value=] contains a four element [=list=].
-The elements of the [=list=] are equal to components of a unit quaternion representing orientation
-of the device in a three dimensional Cartesian coordinate system (x, y, z), where x-axis points east,
-y-axis points north and z-axis points up and is perpendicular to the plane made up of x and y axis.
+The elements of the [=list=] are equal to components of a unit quaternion [cos(θ/2), x**sin(θ/2), y*sin(θ/2), z*sin(θ/2)]
```
Comment (pozdnyakov) [inline index.bs]: Could you add a ref for "unit quaternion"?

Comment (pozdnyakov) [inline index.bs]: maybe better `[cos(θ/2), Vx sin(θ/2), Vy sin(θ/2), Vz sin(θ/2)]`, where `V` is the unit vector representing the axis of rotation

**Comment by anssiko** [comment]
LGTM. Bonus points for referencing my favourite paper.

File: index.bs [hunk 2]
```diff
@@ -70,11 +105,23 @@ The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientat
...
 
-A [=latest reading=] per [=sensor=] of orientation type includes one [=map/entries=]
+A [=latest reading=] per [=sensor=] of orientation type includes an [=map/entry=]
 whose [=map/key=] is "quaternion" and whose [=map/value=] contains a four element [=list=].
-The elements of the [=list=] are equal to components of a unit quaternion representing orientation
-of the device in a three dimensional Cartesian coordinate system (x, y, z), where x-axis points east,
-y-axis points north and z-axis points up and is perpendicular to the plane made up of x and y axis.
+The elements of the [=list=] are equal to components of a unit quaternion [[QUATERNIONS]]
+[cos(θ/2), V<sub>x</sub> * sin(θ/2), V<sub>y</sub> * sin(θ/2), V<sub>z</sub> * sin(θ/2)] where V is the unit vector representing
+the axis of rotation. The quaternion represents rotation of a device hosting motion sensors in relation to a <dfn>Earth's reference coordinate system</dfn> defined as a three
```
Comment (kenchris) [inline index.bs]: Maybe say that the first component is the scalar representing the actual rotation angle

File: index.bs [hunk 3]
```diff
@@ -70,11 +105,23 @@ The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientat
...
+dimensional Cartesian coordinate system (x, y, z), where:
+
+ - x-axis is a vector product of y.z that is tangential to the ground and points east.
+ - y-axis is tangential to the ground and points towards magnetic north.
+ - z-axis points towards the sky and is perpendicular to the plane made up of x and y axes.
```
Comment (kenchris) [inline index.bs]: So the z-axis would normally be called a normal vector of that plane

File: index.bs [hunk 4]
```diff
@@ -18,18 +18,53 @@ Version History: https://github.com/w3c/orientation-sensor/commits/gh-pages/inde
 Indent: 2
```
Comment (kenchris) [inline index.bs]: the perspective of the phone sides on that picture in comparison with the earth, looks all wrong. The phone looks like it lays flat on the surface, but the sides tell another story... the sides should be parallel with the z axis

Comment (alexshalamov) [inline index.bs]: I can improve it later if required, or you can open issue or PR. Thanks.

**Pull request merged**